airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -218,6 +229,7 @@ def __init__(
         self.max_attempts = max_attempts or 60
         self.max_ingestion_time = max_ingestion_time
         self.deferrable = deferrable
+        self.processing_job: dict[Any, Any] | None = None
```

Comment:
Idea here was to have the same structure that is returned to XCom, maybe without real reason to.

Reply:
if you want a dict like this to be returned at the end, then you should create it where you return it, otherwise it just makes the code more confusing to read

Reply:
Done.